src/components/funnels/funnels/register/steps/contact.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsxImportSource @emotion/react */
```

Comment:
We shouldn't have to use `@jsxImportSource` anymore :3

Reply:
Correct!